### ISSUE TITLE
Remove 'TLC' mailing list, and all references

### ISSUE THREAD

**Issue by amarts** [gluster/project-infrastructure#53]
The 'TLC' (Technical Leadership Counsel) is no more active, and I don't foresee any possible resurrection of that, at least in few years. Would be good to remove that list and all references.

FYI: https://lists.gluster.org/mailman/listinfo/tlc

**Comment by amarts** [comment]
@mscherer FYI

**Comment by mscherer** [comment]
yup, I did mask the list on the server. However, while checking ansible, I did see that privacy@gluster go to the tlc. I guess we might need to rethink that part.
